### DISCUSSION TITLE
Update Actian entry in External Dialects table

### DISCUSSION THREAD

**Discussion by hab6** [sqlalchemy/sqlalchemy#11013] [Usage Questions]
The description and link for the **Actian** entry in the **External Dialects** [table](https://github.com/sqlalchemy/sqlalchemy/blob/main/doc/build/dialects/index.rst#external-dialects) needs to be updated.

From:

Database | Dialect
-- | --
Actian Avalanche, Vector, Actian X, and Ingres | [sqlalchemy-ingres](https://github.com/clach04/ingres_sa_dialect)

To:

Database | Dialect
-- | --
Actian Data Platform, Vector, Actian X, Ingres | [sqlalchemy-ingres](https://github.com/ActianCorp/sqlalchemy-ingres)

I work for Actian and am responsible for helping to maintain the [sqlalchemy-ingres](https://github.com/ActianCorp/sqlalchemy-ingres) code, though am not a SQLAlchemy contributor.

Please advise on how to handle this simple repository documentation change. Should I open a pull request?

@clach04

**Comment by CaselIT** [comment]
Hi,

Sure feel free to just send a PR. Will merge it shortly

**Comment by hab6** [comment]
PR #11014 created.

**Comment by hab6** [comment]
Closing since PR #11014 was merged on Feb 16, 2024.
